README.md

```diff
@@ -79,15 +79,18 @@ People
 * [Debarun Dhar](people/debarun-dhar.md) - dd599 - [http://dd599.s3-website-us-east-1.amazonaws.com/grading.html](http://dd599.s3-website-us-east-1.amazonaws.com/grading.html)
+* [Saliou Diallo](people/saliou-diallo.md) - sdd65 - [http://sdd65.com.s3-website-us-east-1.amazonaws.com/grading.html](http://sdd65.com.s3-website-us-east-1.amazonaws.com/grading.html) [![CircleCI](https://circleci.com/gh/sddioulde/skeleton.svg?style=svg)](https://circleci.com/gh/sddioulde/skeleton)
 * [Jessie Gao](people/jessie-gao.md) - jg2238 - [https://s3-us-west-2.amazonaws.com/jessiegao-cs5356.com/grading.html](https://s3-us-west-2.amazonaws.com/jessiegao-cs5356.com/grading.html)
 * [Ethan Green](people/ethan-green.md) - erg53 - [http://erg53.s3-website-us-east-1.amazonaws.com/grading.html](http://erg53.s3-website-us-east-1.amazonaws.com/grading.html)
+* [Zeeshan Hanif](people/zeeshan-hanif.md) - zh278 - [https://s3.us-east-2.amazonaws.com/ctcs53561/grading.html](https://s3.us-east-2.amazonaws.com/ctcs53561/grading.html)
+* [I Huang](people/i-huang.md) - ih265 - [http://ih265.s3-website-us-east-1.amazonaws.com/grading.html](http://ih265.s3-website-us-east-1.amazonaws.com/grading.html)
 * [Zheyuan Gu](people/zheyuan-gu.md) - zg245 - [http://aws-website-guccigu-59k8u.s3-website-us-east-1.amazonaws.com/grading.html](http://aws-website-guccigu-59k8u.s3-website-us-east-1.amazonaws.com/grading.html)
-* [Saliou Diallo](people/saliou-diallo.md) - sdd65 - [http://sdd65.com.s3-website-us-east-1.amazonaws.com/grading.html](http://sdd65.com.s3-website-us-east-1.amazonaws.com/grading.html)
 * [Yiyuan Feng](people/yiyuan-feng.md) - yf323 - [http://yeefeng.me.s3-website.us-east-2.amazonaws.com/grading.html](http://yeefeng.me.s3-website.us-east-2.amazonaws.com/grading.html)
 * [David Hachuel](people/david-hachuel.md) - dh649 - [http://aws-website-startup-systems-dh-ns6pf.s3-website-us-east-1.amazonaws.com/grading.html](http://aws-website-startup-systems-dh-ns6pf.s3-website-us-east-1.amazonaws.com/grading.html) [![CircleCI](https://circleci.com/gh/dhachuel/skeleton.svg?style=svg&circle-token=a4f530a9090ad12cdf99965817cef1b9db2c2fbb)](https://circleci.com/gh/dhachuel/skeleton)
 * [Xu Han](people/xu-han.md) - xh325 - [https://s3.us-east-2.amazonaws.com/leohuntnyc.com/grading.html](https://s3.us-east-2.amazonaws.com/leohuntnyc.com/grading.html)
 * [Zeeshan Hanif](people/zeeshan-hanif.md) - zh278 - [https://s3.us-east-2.amazonaws.com/ctcs53561/grading.html](https://s3.us-east-2.amazonaws.com/ctcs53561/grading.html) [![CircleCI](https://circleci.com/gh/zeeshanh/skeleton.svg?style=svg)](https://circleci.com/gh/zeeshanh/skeleton)
 * [I Huang](people/i-huang.md) - ih265 - [http://ih265.s3-website-us-east-1.amazonaws.com/grading.html](http://ih265.s3-website-us-east-1.amazonaws.com/grading.html) [![CircleCI](https://circleci.com/gh/mikemike07/skeleton.svg?style=svg)](https://circleci.com/gh/mikemike07/skeleton)
+>>>>>>> 025603c5a8b62c1e1e01c51178bc8d42ed18b3fd
```

Comment:
What's this?   remove it please

Reply:
Hmm I must have not noticed. Would it be a good idea to fork your current version again and just add my CircleCI badge?